fastly/user.go

```diff
@@ -13,6 +13,7 @@ type User struct {
 	DeletedAt              *time.Time `mapstructure:"deleted_at"`
 	EmailHash              *string    `mapstructure:"email_hash"`
 	LimitServices          *bool      `mapstructure:"limit_services"`
+	LimitWorkspaces        *bool      `mapstructure:"limit_workspaces"`
```

Comment:
I was checking the customer API https://www.fastly.com/documentation/reference/api/account/customer/ and I don't see the `limit_workspaces` field documented. It's not a blocker for this PR but it's usually a good idea to make sure the docs are updated to reflect the latest interface.

Reply:
@Integralist thank you for reviewing this! We're planning to update the API docs after we release this change